application_form/tests/conftest.py

```diff
@@ -1,3 +1,38 @@
 import faker.config
+from elasticsearch.helpers.test import get_test_client, SkipTest
+from elasticsearch_dsl.connections import add_connection
+from pytest import fixture, skip
+from rest_framework.test import APIClient
+from time import sleep
+
+from connections.tests.factories import ApartmentMinimalFactory
 
 faker.config.DEFAULT_LOCALE = "fi_FI"
+
+
+@fixture(scope="session")
+def api_client():
+    return APIClient()
+
+
+@fixture(autouse=True, scope="session")
+def elastic_client():
+    try:
+        connection = get_test_client()
+        add_connection("default", connection)
+        yield connection
+        connection.indices.delete("test-*", ignore=404)
+    except SkipTest:
+        skip()
+
+
+@fixture(scope="session")
+def elastic_apartments():
+    elastic_apartments = ApartmentMinimalFactory.build_for_sale_batch(10)
+    try:
+        for item in elastic_apartments:
+            item.save()
+        sleep(3)
```

Comment:
Add comment that why here is sleep needed

Reply:
Good point! Actually, I found a better way:
```python
item.save(refresh="wait_for")
```
Which doesn't require sleep, and doesn't seem to make the tests any slower in this particular case.